R/seq_linter.R

```diff
@@ -14,19 +14,22 @@
 #' @export
 seq_linter <- function() {
   bad_funcs <- c("length", "n", "nrow", "ncol", "NROW", "NCOL", "dim")
-  bad_func_xpath <- glue::glue(
+  bad_func_xpath_with_seq <- glue::glue(
+    "expr[1][SYMBOL_FUNCTION_CALL[text() = 'seq']]/following::expr[1]/expr[SYMBOL_FUNCTION_CALL[{xp_text_in_table(bad_funcs)}]]"
```

Comment:
I am usually doing this:

//node
/successor
/successor

Reply:
Thanks, done!